py/kubeflow/testing/ci/kf_unittests.py

```diff
@@ -0,0 +1,15 @@
+"""Test Argo workflow"""
```

Comment:
Can you expand the comment please.

Most people will be a bit confused by this file because the filename says unittests but this doesn't look like a unittest file.

Reply:
Fixed.